src/Campaigns/Actions/AssociateCampaignPageWithCampaign.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Give\Campaigns\Actions;
+
+use Exception;
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Models\CampaignPage;
+
+class AssociateCampaignPageWithCampaign
```

Comment:
Can we add a since tag

Reply:
Resolved https://github.com/impress-org/givewp/pull/7721/commits/9e375e3f10a5f58accfda9028943426552af54c8